app/ui/src/main/java/de/mm20/launcher2/ui/launcher/search/location/LocationItem.kt

```diff
@@ -107,11 +98,164 @@ fun LocationItem(
+                        )
+                    }
+                }
+            }
+            AnimatedVisibility(showDetails) {
+
+                // schedule
+                // rows then cols
+                Column {
+                    if (!openingHours.value.isNullOrEmpty()) {
+                        for ((dow, hours) in openingHours.value!!.groupBy { it.dayOfWeek }) {
+                            Text(
+                                modifier = Modifier.align(Alignment.CenterHorizontally),
+                                text = "$dow: ${hours.joinToString(separator = " ") { "${it.startTime} - ${it.startTime + it.duration}" }}",
+                                style = MaterialTheme.typography.bodySmall,
+                            )
+                        }
+                    }
+
+                    val showMap by viewModel.showMap.collectAsState()
+                    if (showMap) {
+
+                        val zoomLevel = 19
+                        val nTiles = 4
+
+                        val tileServerUrl by viewModel.mapTileServerUrl.collectAsState()
+                        val shape = MaterialTheme.shapes.small
+
+                        MapTiles(
+                            modifier = Modifier
+                                .padding(top = 8.dp, bottom = 8.dp)
+                                .align(Alignment.CenterHorizontally)
+                                .fillMaxSize(.8f)
+                                .aspectRatio(1f)
+                                .border(1.dp, MaterialTheme.colorScheme.outline, shape)
+                                .clip(shape),
+                            tileServerUrl = tileServerUrl,
+                            location = location,
+                            zoomLevel = zoomLevel,
+                            numberOfTiles = nTiles,
+                            userLocation = userLocation?.let { it.latitude to it.longitude },
+                        )
+                    }
+
+                    Text(
+                        modifier = Modifier.align(Alignment.CenterHorizontally),
+                        text = getLocationSummary(
+                            street.value,
+                            houseNumber.value,
+                        ),
+                        style = MaterialTheme.typography.bodySmall
+                    )
+
+                    Toolbar(
+                        modifier = Modifier.fillMaxWidth(),
+                        leftActions = listOf(
+                            DefaultToolbarAction(
+                                label = stringResource(id = R.string.menu_back),
+                                icon = Icons.Rounded.ArrowBack
+                            ) {
+                                onBack()
+                            }
+                        ),
+                        rightActions = listOfNotNull(
+                            websiteUrl.value.runCatching {
```

Comment:
What are you catching here? `Uri.parse`? I think that only ever throws when the provided string is `null` (in which case a simple `?.let` would do the trick)